Sources/Common/NetowrkStruct.swift

```diff
@@ -1,21 +1,23 @@
 import Foundation
 
-struct Struct: Equatable {
-    var fields: [String: StructValue]
+public struct NetworkStruct: Equatable {
+    public init(fields: [String: NetworkStructValue]) {
+        self.fields = fields
+    }
+    public var fields: [String: NetworkStructValue]
 }
 
-enum StructValue: Equatable {
+public enum NetworkStructValue: Equatable {
```

Comment:
This is basically a JsonObject now! Can we re-use something from std and delete this completely?